### DISCUSSION TITLE
Compile error:   Ubuntu 22.04 gcc -m unrecognized command-line option -m and other missing

### DISCUSSION THREAD

**Discussion by rohrbachger** [ipxe/ipxe#1055] [General]
On my new Ubuntu 22.04 the compiling for ipxe does not work anymore
It`s out of the box installed.

Do I`m missing some packages? Google did not delivered simple answers, but I can´t believe that this is really complicate.
Any hints?

root@Desktop:/GRPROG/ipxe/src# gcc --version
gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0

root@Desktop:/GRPROG/ipxe/src# make bin/undionly kpxe EMBED=MyHome.ipxe
gcc -m elf_i386 -N --no-check-sections   --gc-sections -static -z separate-code -z max-page-size=4096  bin/undionly.o   -o bin/undionly
gcc: error: unrecognized command-line option ‘-m’
gcc: error: unrecognized command-line option ‘--no-check-sections’; did you mean ‘--no-data-sections’?
gcc: error: unrecognized command-line option ‘--gc-sections’; did you mean ‘--data-sections’?

**Comment by NiKiZe** [comment]
Is this with current git master of iPXE?
Have you opened any bug towards Ubuntu or GCC for removing used options?

**Comment by rohrbachger** [comment]
Yes, with git master.
No bug opened, at first I want to make sure the problem is not in front of the keyboard.
Maybe I have to install additional tools .

**Comment by rohrbachger** [comment]
Well, I think I fixed with a reboot. 
On a second try that worked like a charm
